docs/reference/cli_env.md

```diff
@@ -89,7 +89,17 @@ from resolved databases.
     Examples: `postgresql://pgwatch@localhost:5432/metrics`,  
     `prometheus://localhost:9090`,  
     `jsonfile:///tmp/metrics.json`,  
-    `rpc://localhost:5000/`
+    `grpc://localhost:5000/`
+
+    gRPC sink supports:
+
+    - **Authentication**: via username/password provided in the URI using the standard format, e.g.:
+    `grpc://pgwatch:pgwatch@localhost:5000/`
+
+    - **TLS encryption**: by specifying a custom Certificate Authority (CA) file via `sslrootca` URI parameter, e.g.:
+    `grpc://localhost:5000/?sslrootca=/home/user/ca.crt`
+
+
```

Comment:
```suggestion
    `grpc://user:pwd@localhost:5000/?sslrootca=/home/user/ca.crt`


```

We don't expose here other sinks parameters, so we shouldn't expose grpc as well.

Reply:
Maybe we need a dedicated pages for every sink parameters and settings